### PR TITLE
No PHP File Functions

### PR DESCRIPTION
**Pull request by khacoder**
Issue #71

### CODE REVIEW

**Comment by grappler** [comment]
Looks good, just needs to pass the check. I need to test it too.

**Comment by jrfnl** [comment]
This looks like duplicate code with upstream. Please check if the upstream code can be/needs to be improved and do so and just refer the upstream sniff in the ruleset.

This should not be a theme specific sniff.

**Comment by grappler** [comment]
I will look that all of the functions are fully integrated in https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/633

Leaving this open for the moment till the PR is merged upstream.

File: WordPress-Theme/ruleset.xml [hunk 1]
```diff
@@ -99,4 +99,12 @@
 		<message>eval() is a security risk so not allowed.</message>
 	</rule>
 
+	<!--
+		Use WordPress functionality and features first, if available.
+		https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features
+	-->
+	<rule ref="WordPress.WP.AlternativeFunctions.file_system_read">
```
Comment (jrfnl) [inline WordPress-Theme/ruleset.xml]: The error codes have changes with WPCS 0.11.0, so using the group name will no longer work.
You will need to add the error code for each function individually now.

The new codes are (hoping I remember correctly by heart):
* `WordPress.WP.AlternativeFunctions.file_system_read_readfile`
* `WordPress.WP.AlternativeFunctions.file_system_read_fopen`
* `WordPress.WP.AlternativeFunctions.file_system_read_sockopen`
* `WordPress.WP.AlternativeFunctions.file_system_read_pfsockopen`
* `WordPress.WP.AlternativeFunctions.file_system_read_fclose`
* `WordPress.WP.AlternativeFunctions.file_system_read_fread`
* `WordPress.WP.AlternativeFunctions.file_system_read_fwrite`
* `WordPress.WP.AlternativeFunctions.file_system_read_file_put_contents`
* `WordPress.WP.AlternativeFunctions.file_system_read_file_get_contents`

**Comment by jrfnl** [comment]
Closing this PR.

A new PR will be opened to add the complete `WP.AlternativeFunctions` sniff to TRTCS.
This will basically be a sniff extending the upstream one and overruling the `getGroups()` method to change the error type for all but two groups.

**Pull request closed without merging**